### DISCUSSION TITLE
Inserting relationship value in table during an automation

### DISCUSSION THREAD

**Discussion by SICUFI** [Budibase/budibase#7772] [Help] [answered]
Hello,

I'm new to Budibase and I'm trying to create an application using Budibase DB where employees can request holidays to their managers.

Basically I've created a table where each employee is listed and another table where each manager is listed, the two are connected by a relationship column in each of these tables. The relationship is one > many since a manager can have more than 1 employee.

There is a third table which stores the Requests. Here we have 2 relationships columns , 1 for each of the other tables.

When a user make a requests I would like to populate those 2 columns automatically, so provided that I know who the user is thanks to the binding CurrentUser.Email i've created a form where I only asks for relevant info and then the "Save" button triggers an automation, which works like this.

![image](https://user-images.githubusercontent.com/113527716/190166073-82055f51-23fe-4fb3-8bd3-b68653f8625b.png)

First it passes the values from the form to the automation

![image](https://user-images.githubusercontent.com/113527716/190166168-090a5ca4-18ec-438a-b3cd-68a7e8b9ab8f.png)

Then it queries the Employee table to match the user (with the email in define filters) and see who is his manager

![image](https://user-images.githubusercontent.com/113527716/190166423-e82379f1-24c2-4920-bf83-56f96fedcf82.png)

Then it stores everything in the Requests table.

The problem I'm facing is that the query where I'm looking for the Manager relationship ("Capo" column) seems to be working as I can see that it correctly returns the value (as an _id and primary display of the relationship) , but when in the third step I try to pass that value to the Requests table it doesn't work and it just creates the row with an empty value.

Can someone please point me in the right direction to resolve this?

Thank you!

**Comment by SICUFI** [comment, marked as answer]
I managed to get it working literally 2 mins after posting this.

For future reference if someone stumbles in this post the solution was to pass the _id of the relationship and not the relationship "as a whole".

So the binding in the third step would be : {{ steps.1.rows.0.Capo.0._id }}

**Reply by Bergem93** [reply]
Hi,

I'm trying to do the the exact same thing as you, fetching data from a Relationship table and create a new row with that data. I've tried your binding, and many variations of it, but I only get an empty field. 

Could you check my post and see if you notice anything that would solve it?https://github.com/Budibase/budibase/discussions/9710
